hikari/files.py

```diff
@@ -986,6 +1054,11 @@ def _assert_bytes(data: typing.Any) -> bytes:
         return data
 
 
+def _write_bytes(path: Pathish, default_filename: str, force: bool, data: bytes) -> None:
```

Comment:
```suggestion
def _write_bytes(
    path: Pathish, default_filename: str, force: bool, data: typing.Union[bytearray, bytes, memoryview]
) -> None:
```
Forgot this line needs to be updated as well

Reply:
Nice catch!